paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h

```diff
@@ -91,8 +132,44 @@ class FusedEmbeddingSeqPoolKernel : public framework::OpKernel<T> {
     output_t->Resize({batch_size, last_dim});
 
     if (combiner_type == "sum") {
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
```

Comment:
is this optimization specifically for Sparse matrix ?

If so, then the macro condition seems not fair.

Reply:
Since the data of input tensor should be specified, it will statics the count of data index. then the count will be multiplied by weights and got the sum. Actually it is not only for sparse matrix.  From anther side, perhaps the input data is sparse in more scenes .  you can think that sparse matrix be one special way to express the dense matrix.

**k** : batch size
**m**: rows of weights
**n**: clos of weights
**A**: The matrix of index count (to enumerate the content of input tensor)
**B**: Weights
**C**: Output

**A** k x m * **B** m x n = **C** k x n

Reply:
is this only enabled on linux? reason? only has linux version mkl lib?

If only for linux, does it impact ut?

Reply:
so if input data is spare, then performance gonna be good, but if it is not, then performance gets bad or still be better than before? or you think in emb_seq_pool case, input should always be spare matrix?

Reply:
@LeoZhao-Intel For Windows, we need rebuild the mklml library for windows. and until now there is not to support for MAC and IOS in mklml.cmake file.  It can not impact ut of this operator.
In emb_seq_pool case, the input data should be always be spare because the input lengths be more less than the rows of weights.

Reply:
@LeoZhao-Intel @yihuaxu We could support Linux at first.

Reply:
It's fine here, but we should consider further more.